File: spark-beer-api/src/main/java/com/example/spark/beer/api/service/BarService.java
package com.example.spark.beer.api.service;


import com.example.spark.beer.api.entity.Order;
import lombok.extern.slf4j.Slf4j;

import java.util.UUID;

@Slf4j
public class BarService {
    public String makeOrder(Order order) {
        log.debug("Order {} accepted", order);
        return UUID.randomUUID().toString();
    }
}

File: kofu-beer-api/src/main/kotlin/com/example/kofu/beer/api/service/BarService.kt
package com.example.kofu.beer.api.service

import com.example.kofu.beer.api.entity.Order
import org.slf4j.LoggerFactory
import java.util.*

class BarService {
    val Log = LoggerFactory.getLogger(BarService::class.java.name)
    fun makeOrder(order: Order): String {
        Log.debug("Order {} accepted", order)
        return UUID.randomUUID().toString()
    }
}